src/metadata/exiv2_handler.h

```diff
@@ -36,12 +36,16 @@
 
 #include "metadata_handler.h"
 
+#include <exiv2/exiv2.hpp>
+
 /// \brief This class is responsible for reading exif header metadata
 class Exiv2Handler : public MetadataHandler {
 public:
     explicit Exiv2Handler(const std::shared_ptr<Context>& context)
         : MetadataHandler(context)
     {
+        // silence annoying and pointless warnings
+        Exiv2::LogMsg::setLevel(Exiv2::LogMsg::mute);
     }
```

Comment:
We should make that depend on gerbera debug mode. Best to split the change into separate PR.

Reply:
That's actually here: https://github.com/gerbera/gerbera/pull/2114 . It leaked into this commit.